packages/squiggle-lang/src/fr/tag.ts

```diff
@@ -185,4 +186,32 @@ export const library = [
       ),
     ],
   }),
+  maker.make({
+    name: "omit",
+    examples: [],
+    definitions: [
+      makeDefinition(
+        [frForceBoxed(frAny({ genericName: "A" })), frArray(frString)],
```

Comment:
It would be nice if we had an frEnum() or a stringDomain here, but it's not too critical right now. Would be interested if you have any quick thoughts on this.